geometry_draw/src/main/scala/myConstructions/test_1.scala

```diff
@@ -0,0 +1,12 @@
+import geometry_draw._
+package myConstructions
```

Comment:
This line (the line with `package`) should be the first line in the file. That's probably what's causing the error.